src/openforms/authentication/tests/test_views.py

```diff
@@ -140,6 +148,44 @@ def test_plugin_start_failure_redirects(self):
         expected.args[BACKEND_OUTAGE_RESPONSE_PARAMETER] = "plugin1"
         self.assertEqual(furl(response["Location"]), expected)
 
+    @override_settings(CORS_ALLOW_ALL_ORIGINS=True)
+    def test_plugin_start_failure_not_enabled(self):
+        register = Registry()
+        register("plugin1")(Plugin)
+        plugin = register["plugin1"]
+
+        config = GlobalConfiguration.get_solo()
+        config.plugin_configuration = {
+            "authentication": {"plugin1": {"enabled": False}}
+        }
+        config.save()
+
+        step = FormStepFactory(
+            form__slug="myform",
+            form__authentication_backends=["plugin1"],
+            form_definition__login_required=True,
+        )
+        form = step.form
+
+        # we need an arbitrary request
+        factory = RequestFactory()
+        init_request = factory.get("/foo")
+
+        # actual test starts here
+        next_url_enc = quote("http://foo.bar?bazz=buzz")
+
+        url = plugin.get_start_url(init_request, form)
+
+        start_view = AuthenticationStartView.as_view(register=register)
+        response = start_view(
+            factory.get(f"{url}?next={next_url_enc}"),
```

Comment:
please don't build querystring params by hand, but in tests rather pass them as a dict so django does the URL encoding for you. And if you have to build/manipulate URLs, use `furl` which is already a dependency of the project.

Reply:
Ah, I was a bit to eager with copying from existing tests again :p